src/components/elements/card.js

```diff
@@ -389,6 +391,7 @@ export const NavCard = ({
 }
 
 NavCard.propTypes = {
+    aria_label: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
```

Comment:
Quick question...do we still need object prop type since we're no longer localising the labels?

Reply:
No we don't. Good catch~